docs/CODING_STYLE.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Coding Style
+category: Contributing
+layout: default
+SPDX-License-Identifier: LGPL-2.1-or-later
+---
+
+# Coding Style
+
+## Python Version
+
+- The lowest supported Python version is CPython 3.9.
+
+## Formatting
+
+- Use the accompanying `.editconfig` or `.dir-locals.el`.
+- For Python files we use the style from `ruff format` with a line length of 109 characters and four spaces
+  indentation.
+- Indentation with tabs is not allowed.
+- When it improves readability, judicious use of `# noqa: E501` comments is allowed.
+- Long lists, including argument lists, should have a trailing comma to force ruff to split all elements on a
+  line of their own.
+- List of commandline arguments should not split the argument of a commandline option and the option. This
+  needs to be enforced with `# fmt: skip` comments, e.g. do
+  ```python
+  cmd = [
+      "--option", "foo",
+  ]  # fmt: skip
+  ```
+  and do NOT
+  ```python
+  cmd = [
+      "--option",
+      "foo",
+  ]
+  ```
+- When coercing Path-like objects to strings, use `os.fspath`, since this calls the `__fspath__` protocol
```

Comment:
Missing period

Reply:
Isn't the period already after `__str__` on the following line?